routes/patch.js

```diff
@@ -104,7 +105,7 @@ function createPatch(tile, geoJson, overviews, dirCache) {
   const PImage = require('pureimage');
   const debugProcess = require('debug')('patch');
   const fsProcess = require('fs');
-  const pathProcess = require('path');
+  // const pathProcess = require('path');
```

Comment:
const rok4 = require('../rok4.js'); manquant ?
J'obtiens une erreur lors de la creation d'un patch :
patch on a reçu une erreur :  ReferenceError: rok4 is not defined
    at Function.createPatch (eval at run (D:\PackO\node_modules\workerpool\src\worker.js:100:11), <anonymous>:57:20)
    at Function.run (D:\PackO\node_modules\workerpool\src\worker.js:101:12)
    at MessagePort.<anonymous> (D:\PackO\node_modules\workerpool\src\worker.js:121:27)
    at MessagePort.[nodejs.internal.kHybridDispatch] (internal/event_target.js:354:41)
    at MessagePort.exports.emitMessage (internal/per_context/messageport.js:18:26)